src/content/1.7/testing/ui-tests/how-to-contribute-and-create-web-acceptance-tests.md

```diff
@@ -1,12 +1,12 @@
 ---
-title: How to contribute and create web acceptance tests
-menuTitle: Creation of web acceptance tests
+title: How to contribute and create UI tests
+menuTitle: Creation of UI tests
 weight: 2
 aliases:
-   - /1.7/testing/how-to-create-your-own-web-acceptance-tests
+   - /1.7/testing/how-to-create-your-own-ui-tests
```

Comment:
```suggestion
   - /1.7/testing/how-to-create-your-own-web-acceptance-tests
```

Reply:
Can't we have some sort of redirection ?

Reply:
If I remember, redirection is for pages which have already created and referenced by BigG. 
Aliases are redirections.

Reply:
Exactly, aliases are the "old urls"  https://gohugo.io/content-management/urls/